src/server/pfs/fuse/filesystem_seek_mac_test.go

```diff
@@ -27,7 +27,7 @@ func TestSeekRead(t *testing.T) {
 	testFuse(t, func(c client.APIClient, mountpoint string) {
 		repo := "test"
 		require.NoError(t, c.CreateRepo(repo))
-		commit, err := c.StartCommit(repo, "", "")
+		commit, err := c.StartCommit(repo, "")
```

Comment:
This will result in an error actually.  The second argument is not allowed to be empty.  Maybe just use "master"?